### PR TITLE
ci: winget

### PR DESCRIPTION
**Pull request by lewisakura**
Based off of Dev Home's Winget CI. `WINGET_PAT` will need to be set with the bot account PAT.

Supersedes and thus closes #98 and closes #100.

### CODE REVIEW

File: .github/workflows/winget-submission.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  winget:
+    name: Publish winget package
+    runs-on: windows-latest
+    steps:
+      - name: Submit package to Winget Community Repo
+        run: |
+
+          $packageId = "Vencord.Vesktop"
+          $gitToken = "${{ secrets.WINGET_PAT }}"
```
Comment (Vendicated) [inline .github/workflows/winget-submission.yml:17]: isn't this kinda bad? I'm fairly certain best practice is instead using the env: option to pass the secret as an environment variable

Comment (lewisakura) [reply .github/workflows/winget-submission.yml:17]: I don't think this matters, but I do prefer the environment variable way. I've quite literally just taken this from [Microsoft's own workflow](https://github.com/microsoft/devhome/blob/944975f98a173da6e03d0ae2cf49b23d861e454e/.github/workflows/winget-submission.yml), so.

**Review by Vendicated — APPROVED** [review]
LSTM

**Comment by Vendicated** [comment]
(u can merge if u wanna)

**Comment by lewisakura** [comment]
(i dont have merge perms here silly)

**Comment by Vendicated** [comment]
wtf!!!!!

**Comment by nyakowint** [comment]
dies of crigne

**Pull request merged**